Author: Andreas Tille <tille@debian.org>
Last-Update: Sat, 19 Feb 2022 11:53:03 +0100
Description: Upstream creates static library with versioning / linking scheme
 which looks as it would be intended to create a shared library.  This patch
 really created this shared library (which ends up in a seprate binary package)

--- a/libs/align/CMakeLists.txt
+++ b/libs/align/CMakeLists.txt
@@ -83,6 +83,7 @@ GenerateStaticLibs( align-access "${ALIG
 
 GenerateStaticLibs( sam-extract "${SAM_EXTRACT_SRC}" )
 ExportStatic( sam-extract true )
+ExportShared( sam-extract true "" )
 
 GenerateStaticLibs( align-reader "${ALIGN_READER_SRC}" )
 
